scons/site_tools/stm32_parse_headers.py

```diff
@@ -0,0 +1,48 @@
+import re
```

Comment:
Add a BSD licence header with your name, my name and Sascha's name if he contributed. I give permission to relicense my contributions to this code as BSD.